hadoop-ozone/dist/src/shell/ozone/start-ozone.sh

```diff
@@ -82,7 +82,8 @@ OZONE_JUMBO_RETCOUNTER=$?
 
 #---------------------------------------------------------
 # Ozone ozonemanager nodes
-OM_NODES=$("${OZONE_HOME}/bin/ozone" getconf -ozonemanagers 2>/dev/null)
+TMP_OM_NODES=$("${OZONE_HOME}/bin/ozone" getconf -ozonemanagers 2>/dev/null)
+OM_NODES=`echo $TMP_OM_NODES | grep -o '\([0-9]\{1,3\}\.\)\{3\}[0-9]\{1,3\}' | sort -d | uniq`
```

Comment:
Please add a comment for what the intent of this line is.

Reply:
Ok, thanks for the tip.